management/src/test/resources/clusteredConfiguration.txt

```diff
@@ -0,0 +1,24 @@
+caches:
+    cache-1:
+        keyType: java.lang.String
+        valueType: java.lang.String
+        serviceConfigurations:
+        evictionAdvisor:
```

Comment:
hum, interesting thoughts... I'm not sure it makes sense to display only non null values.
Anybody else cares to chime in ?
If that would be the case, then I guess I'd need to make sure all null values keys are not displayed